Tasks/AzureResourceManagerTemplateDeploymentV3/operations/DeploymentScopeBase.ts

```diff
@@ -68,6 +69,14 @@ export class DeploymentScopeBase {
         }
     }
 
+    private getAzurePortalDeploymentURL() {
+        let portalUrl = this.taskParameters.endpointPortalUrl ? this.taskParameters.endpointPortalUrl : "https://portal.azure.com";
+        portalUrl += "/#blade/HubsExtension/DeploymentDetailsBlade/overview/id/";
+
+        let subscriptionSpecificURL = "/subscriptions/" + this.taskParameters.subscriptionId + "/resourceGroups/" + this.taskParameters.resourceGroupName + "/providers/Microsoft.Resources/deployments/" + this.taskParameters.deploymentName;
```

Comment:
I think subscription and resource group name can be empty if deployment scope is MG or subscription level. Please handle those cases.

Reply:
I've handled it, Thanks! In case of MG,  I couldn't find any deployment link to show that can gather the details.